workflow/src/image_generation.py

```diff
@@ -18,6 +18,8 @@
 import openai
 
 openai.api_key = os.getenv("api_key")
+if os.getenv("custom_api_url"):
```

Comment:
`custom_api_url` should also be set using the workflows configuration. Please add an additional field, a small documentation and add it to [this](https://github.com/chrislemke/ChatFred#configure-the-workflow-optional-) part of the docs. Thx!!

Reply:
Hi, I've added this in an additional field and updated the docs.